rxbinding-design/src/androidTest/java/com/jakewharton/rxbinding/support/design/widget/RxTextInputLayoutTest.java

```diff
@@ -0,0 +1,44 @@
+package com.jakewharton.rxbinding.support.design.widget;
+
+import android.content.Context;
+import android.support.design.widget.TextInputLayout;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.annotation.UiThreadTest;
+import android.support.test.rule.UiThreadTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.view.ContextThemeWrapper;
+import com.jakewharton.rxbinding.support.design.test.R;
+import java.lang.reflect.Field;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static com.google.common.truth.Truth.assertThat;
+
+@RunWith(AndroidJUnit4.class)
+public class RxTextInputLayoutTest {
+  @Rule public final UiThreadTestRule uiThread = new UiThreadTestRule();
+
+  private final Context rawContext = InstrumentationRegistry.getContext();
+  private final Context context = new ContextThemeWrapper(rawContext,
+      com.jakewharton.rxbinding.support.design.R.style.Theme_AppCompat);
+  private final TextInputLayout view = new TextInputLayout(context);
+
+  @Test @UiThreadTest public void counterEnable()
+      throws NoSuchFieldException, IllegalAccessException {
+    RxTextInputLayout.counterEnable(view).call(true);
+    Field field = view.getClass().getDeclaredField("mCounterEnabled");
```

Comment:
:cry: 

Add `// TODO replace with getter once http://b.android.com/191175 is fixed.`